src/Sql/Sql/Elastic Jobs/Cmdlet/JobExecution/GetAzureSqlElasticJobExecution.cs

```diff
@@ -0,0 +1,292 @@
+            Mandatory = true,
+            Position = 0,
+            HelpMessage = "The resource group name.")]
+        [Parameter(ParameterSetName = GetRootJobExecution,
+            Mandatory = true,
+            Position = 0,
+            HelpMessage = "The resource group name.")]
+        [ValidateNotNullOrEmpty]
+        [ResourceGroupCompleter]
+        public override string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the agent object input model
+        /// </summary>
+        [Parameter(
+            ParameterSetName = InputObjectParameterSet,
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The job execution id.")]
+        [Parameter(
+            ParameterSetName = GetRootJobExecutionByParentObject,
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The job execution id.")]
+        [ValidateNotNullOrEmpty]
+        public AzureSqlElasticJobAgentModel ParentObject { get; set; }
+
+        /// <summary>
+        /// Gets or sets the agent resource id
+        /// </summary>
+        [Parameter(
+            ParameterSetName = ResourceIdParameterSet,
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The agent resource id.")]
+        [Parameter(
+            ParameterSetName = GetRootJobExecutionByParentResourceId,
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The agent resource id.")]
+        [ValidateNotNullOrEmpty]
+        public string ParentResourceId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the server name
+        /// </summary>
+        [Parameter(ParameterSetName = DefaultParameterSet,
+            Mandatory = true,
+            Position = 1,
+            HelpMessage = "The server name.")]
+        [Parameter(ParameterSetName = GetRootJobExecution,
+            Mandatory = true,
+            Position = 1,
+            HelpMessage = "The server name.")]
+        [ValidateNotNullOrEmpty]
+        public override string ServerName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the agent name
+        /// </summary>
+        [Parameter(ParameterSetName = DefaultParameterSet,
+            Mandatory = true,
+            Position = 2,
+            HelpMessage = "The agent name.")]
+        [Parameter(ParameterSetName = GetRootJobExecution,
+            Mandatory = true,
+            Position = 2,
+            HelpMessage = "The agent name.")]
+        [ValidateNotNullOrEmpty]
+        public override string AgentName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the job name
+        /// </summary>
+        [Parameter(ParameterSetName = DefaultParameterSet,
+
```

Comment:
@johnpaulkee nit: would you mind cleaning up the empty lines for this parameter?

Reply:
👍